### PR TITLE
#452 Adding build workflow

### PR DESCRIPTION
**Pull request by symphony-thibault**
### Ticket
Relates to https://github.com/finos/symphony-bdk-java/issues/452

Adding GH Actions in replacement of CircleCI.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        # 11 runtime needed for Sonar, we still build with 8 as a target
+        java-version: 11
+    - name: Grant execute permission for gradlew
```
Comment (symphony-youri) [inline .github/workflows/build.yml:24]: that's weird, issn't it part of git attributes?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```
Comment (symphony-youri) [inline .github/workflows/build.yml:18]: if we want to make it faster we can start to cache dependencies as well

File: .github/workflows/build.yml (hunk 1 above)
Comment (symphony-thibault) [reply .github/workflows/build.yml:23]: I'll try without, I took the template generated with GH itself

File: .github/workflows/build.yml (hunk 2 above)
Comment (symphony-thibault) [reply .github/workflows/build.yml:18]: OK I'll follow https://github.com/symphonyoss/messageml-utils/blob/master/.github/workflows/build.yml#L16

Comment (symphony-thibault) [reply .github/workflows/build.yml:18]: 👍  done below

**Pull request merged**